sheets_and_friends/tsv_in/modifications_long.tsv

```diff
@@ -208,12 +208,11 @@ SoilInterface	slope_aspect	overwrite_examples	examples			35 degrees
 SoilInterface	slope_aspect	add_attribute	todos			This is no longer matching the listed IRI from GSC. When NMDC has its own slots, map this to the MIxS slot
 SoilInterface	slope_gradient	replace_attribute	string_serialization			{percentage}
 SoilInterface	slope_gradient	overwrite_examples	examples			10%
-DhMultiviewCommonColumnsMixin	source_mat_id	add_attribute	comments			Identifiers must be prefixed. Possible FAIR prefixes are IGSNs (http://www.geosamples.org/getigsn), NCBI biosample accession numbers, ARK identifiers (https://arks.org/). These IDs enable linking to derived analytes and subsamples. If you have not assigned FAIR identifiers to your samples, you can generate UUIDs (https://www.uuidgenerator.net/).
-DhMultiviewCommonColumnsMixin	source_mat_id	replace_attribute	description			A globally unique identifier assigned to the biological sample.
-DhMultiviewCommonColumnsMixin	source_mat_id	overwrite_examples	examples			IGSN:AU1243|UUID:24f1467a-40f4-11ed-b878-0242ac120002
-DhMultiviewCommonColumnsMixin	source_mat_id	add_attribute	notes			The source material IS the Globally Unique ID
+DhMultiviewCommonColumnsMixin	source_mat_id	add_attribute	comments			Identifiers must be prefixed. Possible FAIR prefixes are: `igsn` for International Generic Sample Numbers (http://www.geosamples.org/getigsn), `biosample` for NCBI biosample accession IDs, `gold` for GOLD identifiers.
+DhMultiviewCommonColumnsMixin	source_mat_id	replace_attribute	description			A globally unique identifier assigned to the parent sample or sample that is the source of this sample.
+DhMultiviewCommonColumnsMixin	source_mat_id	overwrite_examples	examples			igsn:AU1243|biosample:SAMEA2397676 
 DhMultiviewCommonColumnsMixin	source_mat_id	replace_attribute	string_serialization			{text}:{text}
-DhMultiviewCommonColumnsMixin	source_mat_id	add_attribute	todos			Currently, the comments say to use UUIDs. However, if we implement assigning NMDC identifiers with the minter we dont need to require a GUID. It can be an optional field to fill out only if they already have a resolvable ID.
+DhMultiviewCommonColumnsMixin	source_mat_id	replace_attribute	pattern			^(igsn:[a-zA-Z0-9]+|biosample:SAMN[a-zA-Z0-9]+|biosample:SAME[a-zA-Z0-9]+|biosample:SAMJ[a-zA-Z0-9]+|gold:Gb[0-9]+)(;(igsn:[a-zA-Z0-9]+|biosample:SAMN[a-zA-Z0-9]+|biosample:SAME[a-zA-Z0-9]+|biosample:SAMJ[a-zA-Z0-9]+|gold:Gb[0-9]+))*$
```

Comment:
@turbomam can you take a look at this. Considering the example data that @aclum commented on that gold:012345 shouldn't pass.
I agree... but it looks like it is.. and shouldn't be?

Reply:
@pkalita-lbl - submission-schema build complex has modifications from here & in the validation converter.tsv file. 
This file looks at the string_serialization meta slot and overwerites the pattern!

Add a line here to remove `string_seralization`

Example - SedimentInterface|SoilInterface	microbial_biomass_n	remove_attribute	string_serialization